confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -651,14 +651,51 @@ public TSchemaPartitionTableResp getOrCreateSchemaPartition(PathPatternTree patt
         partitionManager.getOrCreateSchemaPartition(getOrCreateSchemaPartitionPlan);
     resp = queryResult.convertToRpcSchemaPartitionTableResp();
 
-    LOGGER.debug(
-        "GetOrCreateSchemaPartition receive devicePaths: {}, return TSchemaPartitionResp: {}",
-        devicePaths,
-        resp);
+    if (CONF.isPartitionDebugEnabled()) {
+      printNewCreatedSchemaPartition(devicePaths, resp);
+    }
 
     return resp;
   }
 
+  private void printNewCreatedSchemaPartition(
+      List<String> devicePaths, TSchemaPartitionTableResp resp) {
+    StringBuilder devicePathString = new StringBuilder("{");
```

Comment:
Please modify `\n` to `%n` to adapt different system.

Reply:
I've used System.lineSeparator() instead.